### ISSUE TITLE
Why do I get error after install it and configure it in webpack

### ISSUE THREAD

**Issue by eschao** [monoguerin/vue-svg-sprite-loader#2]
Following your doc, I added it in webpack, but still can't import svg correctly, got the below error in browser:

```
[Vue warn]: Invalid Component definition: data:image/svg+xml;base64,bW9kdWxlLmV4cG9ydHMgPSB7dW5kZWZpbmVkfTs=
found in
...
```
Really think using SVG is bad idea in html, but I don't have to choice now. Thanks a lot!

**Comment by monoguerin** [comment]
That means that there is a problem with your webpack, please check that you apply the rule in the right order, meaning that the vue loader doesn't use the url-loader or any other loader to the SVGs

**Comment by eschao** [comment]
@monoguerin I use the default Vue webpack configuration which is generated by Vue framework, I didn't change it before, I just added your recommended two lines in it. I will past my configuration, it maybe a little long

**Comment by monoguerin** [comment]
The webpack template uses the url loader for the svgs, then you have to add the rule of this loader to be run before 

Check the `webpack.base.conf.js` in the build folder in your project

```
{
        test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
        loader: 'url-loader',
        options: {
          limit: 10000,
          name: utils.assetsPath('img/[name].[hash:7].[ext]')
        }
 }
```

you can remove the svg extension there or you can add the rule before that so it is run before that one

**Comment by eschao** [comment]
@monoguerin Thanks, I removed it, but why I got the error as I mentioned before:
```
[Vue warn]: Failed to mount component: template or render function not defined.
```
If I just only import icon and don't use it in <template>, it is ok, If I use it in <template>, the above error will be raised, do you know why? thanks

**Comment by monoguerin** [comment]
I think I found the issue, can you check in your `.svg` sprite if each `<object>` has the property `xmls` ? like

```
<symbol xmlns="http://www.w3.org/2000/svg"...
```

**Comment by eschao** [comment]
@monoguerin yes, you're right, my font file doesn't have this line for symbol. I tried to add it and its ok. You may need to deal it specially.

**Comment by eschao** [comment]
@monoguerin I think you need to handle like this:
```js
    if (!!key && !!viewBox && !!path) {
      var openTag = "<svg "
      if (!!xmlns) {
        openTag += "'" + xmlns + "'"
      }
      openTag += "aria-labelledby='" + key + "' viewBox='" + viewBox + "'>";
```
I tried and it works

**Comment by monoguerin** [comment]
Yeah I already created the PR, for that #3, thanks

**Comment by eschao** [comment]
@monoguerin Ok, looks like it does work, thanks

**Comment by monoguerin** [comment]
@eschao just published version 1.1.0 with the fix for this
